deepchem/models/openvino_model.py

```diff
@@ -0,0 +1,86 @@
+import os
```

Comment:
Could you add some unit tests for the OpenVINO model to make sure it works correctly?

Reply:
Done